app/controllers/application_controller.rb

```diff
@@ -1343,6 +1343,7 @@ def get_view_process_search_text(view)
   end
 
   def perpage_key(dbname)
+    return :reports if dbname == "miqreportresult"
```

Comment:
I don't really like this :disappointed_relieved: could you at least convert it into a case?